native/cocos/bindings/jswrapper/PrivateObject.h

```diff
@@ -131,33 +131,24 @@ class CCIntrusivePtrPrivateObject final : public TypedPrivateObject<T> {
 
 private:
     cc::IntrusivePtr<T> _ptr;
-
     friend TypedPrivateObject<T>;
 };
 
-template <typename T>
-inline typename std::enable_if_t<std::is_destructible<T>::value, void> cctryDelete(T *t) {
-    delete t;
-}
-template <typename T>
-inline typename std::enable_if_t<!std::is_destructible<T>::value, void> cctryDelete(T *t) {
-}
-
```

Comment:
Is these changed related with this PR?

Reply:
These are small changes and descripted in changelog

Reply:
Please do one thing in a PR.

Reply:
fixed